### PR TITLE
Add support for bramble + redfin

### PR DESCRIPTION
**Pull request by 306bobby-android**

### CODE REVIEW

File: bramble.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Bobby (306bobby)",
+			"oem": "",
```
Comment (gwolf2u) [inline bramble.json]: missing OEM

File: bramble.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "",
```
Comment (gwolf2u) [inline bramble.json]: missing device

File: bramble.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"filename": "crDroidAndroid-13.0-20240329-bramble-v9.15.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/bramble/9.x/crDroidAndroid-13.0-20240329-bramble-v9.15.zip/download",
```
Comment (gwolf2u) [inline bramble.json:8]: do upload file

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by 306bobby-android** [comment]
redfin, bramble, and barbet have been published to SF. DT are on org github. All is ready for confirmation

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**